cloud

```diff
@@ -1911,6 +1927,9 @@ function execute_command() {
             --smtp-topic="$smtp_topic" \
             --smtp-subscription="$smtp_subscription"
         sections_run "$sections" firestore_deploy "$project"
+        sections_run "$sections" storage_deploy \
+            --project="$project" \
+            --bucket="$storage_bucket"    
```

Comment:
You're passing these as options, but nothing reads them. This only works because of the weird scoping rules of the shell, where a function called from another function has access to the caller's variables. It exists, but we should never use it, because it makes the code very hard to read, and should pass our arguments explicitly. The same way it's done in other functions. Additionally, we shouldn't be using options for a function with so little arguments. Use positional arguments instead.

Reply:
got it, thank you. I realized it later better after your comment.